src/Analyzers/CSharp/Tests/ForEachCast/ForEachCastTests.cs

```diff
@@ -27,6 +29,7 @@ public class ForEachCastTests
                 [*]
                 dotnet_style_prefer_foreach_explicit_cast_in_source=
                 """ + optionValue,
+                ReferenceAssemblies = ReferenceAssemblies.Net.Net60,
```

Comment:
please don't change the core test.  make it so that some tests use different references assemblies so we can see the different behaviors.